assets/screenshots/amoScreenshots.csv

```diff
@@ -0,0 +1,16 @@
+autocorrectedTextChrome.png, "Show a live preview of all the text transform options directly in the context menu."
+autocorrectedTextFirefox.png, "Show a live preview of all the text transform options directly in the context menu."
+autocorrectedTextThunderbird.png, "Show a live preview of all the text transform options directly in the context menu."
+optionsPageChrome.png, "Disable any features you do not want or just make use of them all. You can adjust the style and behavior as you like!"
```

Comment:
when re-reading/proofreading it: `make use of them all` – uhm is this proper English?

should not it say `make use of all of them`?

Reply:
Yeah, will do.